ajcore.20160503.083804.142.txt

```diff
@@ -0,0 +1,337 @@
+---- AspectJ Properties ---
```

Comment:
minor: remove this file.

It's been on the todo list to fix the output of this file into the target dir, (or short term gitignore).